packages/image-block/src/ImageBlock.tsx

```diff
@@ -28,3 +54,86 @@ export const ImageBlock = ({ appBridge }: BlockProps) => {
         </div>
     );
 };
+
+export const Image = ({
+    url,
+    blockSettings,
+    isEditing,
+    imageName,
+}: {
+    url: string;
+    imageName: string;
+    blockSettings: Settings;
+    isEditing: boolean;
+}) => {
+    const borderRadius = blockSettings.hasRadius_cornerRadius
+        ? blockSettings.radiusValue_cornerRadius
+        : radiusValues[blockSettings.radiusChoice_cornerRadius];
+    const border = blockSettings.hasBorder
+        ? `${blockSettings.borderWidth} ${blockSettings.borderStyle} ${toRgbaString(blockSettings.borderColor)}`
+        : undefined;
+
+    const width =
+        blockSettings.positioning === CaptionPosition.Above || blockSettings.positioning === CaptionPosition.Below
+            ? '100%'
+            : rationValues[blockSettings.ratio];
+    const link = blockSettings.hasLink ? blockSettings.linkObject : undefined;
+
+    const downloadImage = () => {
+        fetch(url).then((response) => {
+            response.blob().then((blob) => {
+                const url = URL.createObjectURL(blob);
+                const a = document.createElement('a');
+                a.href = url;
+                a.download = imageName;
+                a.click();
+            });
+        });
+    };
+
+    const ImageComponent = () => (
+        <div
+            className={joinClassNames([
+                'tw-relative tw-flex tw-h-auto tw-w-full',
+                mapAlignmentClasses[blockSettings.alignment],
+            ])}
+            style={{
+                width,
+                border,
+                padding: blockSettings.hasCustomPadding
+                    ? blockSettings.paddingCustom
+                    : paddingValues[blockSettings.paddingChoice],
+                borderRadius: borderRadius ?? radiusValues[CornerRadius.None],
+                backgroundColor: blockSettings.hasBackground ? toRgbaString(blockSettings.backgroundColor) : undefined,
+            }}
+        >
+            <img src={url} className="tw-w-full" />
```

Comment:
I think with accessibility in mind this should have an `alt` attribute.

Reply:
oh good point! added the alt text.